client/state/data-layer/wpcom/sites/plugins/new/index.js

```diff
@@ -79,14 +77,23 @@ export const uploadComplete =
 		} );
 	};
 
-export const receiveError = ( { siteId }, error ) => [
-	recordTracksEvent( 'calypso_plugin_upload_error', {
-		error_code: error.error,
-		error_message: error.message,
-	} ),
-	showErrorNotice( error ),
-	pluginUploadError( siteId, error ),
-];
+export const receiveError = ( { siteId }, error ) => {
+	const actions = [
+		recordTracksEvent( 'calypso_plugin_upload_error', {
+			error_code: error.error,
+			error_message: error.message,
+		} ),
+		pluginUploadError( siteId, error ),
+	];
+
+	// Mute error messages that are duplicated with content.
```

Comment:
[nit] Would it be better to add the link to the place that shows duplicate content?

Reply:
Do you mean a path to the file or a link to a ticket?

At the very least, I'll improve the messaging.